utils/mock-data-loader/gef/api.js

```diff
@@ -41,12 +43,104 @@ const listApplication = async (token) => {
       Accepts: 'application/json',
       Authorization: token || '',
     },
-    url: `${urlRoot}/v1/gef/application?page=null&pageSize=null`,
+    url: `${urlRoot}/v1/gef/application`, // ?page=null&pageSize=null
```

Comment:
This needs the applicationID to identify which application to delete

Reply:
I believe your looking at the wrong route here

Reply:
As far as I can see, it's the /application/:id that has a delete associated with it? Is it referring to somewhere else?

**portal-api/src/v1/gef/routes.js:**
router.route('/application')
  .get(validate({ role: ['maker', 'checker', 'data-admin'] }), application.getAll)
  .post(validate({ role: ['maker', 'data-admin'] }), application.create);

router.route('/application/:id')
  .get(validate({ role: ['maker', 'checker', 'data-admin'] }), application.getById)
  .put(validate({ role: ['maker', 'data-admin'] }), application.update)
  .delete(validate({ role: ['maker', 'data-admin'] }), application.delete);